Week1/js1-w1-A.A.A.js

```diff
@@ -0,0 +1,175 @@
+// Greetings
+
+console.log ('Hello World!'); //English
+
+console.log ('Halo, Dunia!'); // Indonesian
+
+console.log ('Ciao Mondo!'); // Italian
+
+console.log ('Hola Mundo!'); // Spanish
+
+
+// Debug single quote issue
+
+console.log('I\'m awesome');
+
+// Variables & Values
+
+var x;
+
+console.log("the value of my variable x will be: a number");
+
+console.log(x);
+
+x = 12;
+
+console.log("the value of my variable x will be: a number");
+
+console.log(x);
+
+var y = 'Tom';
+
+console.log("the value of my atring y will be: a name");
+
+console.log(y);
+
+var y = y + '&Jerry';
+
+console.log("the value of my string y will be: a cartoon name");
+
+console.log(y);
+
+var z = 7.25;
+
+console.log(z);
+
+// Round z to the closest integer
+
+var round = Math.round;
+
+var a = round(z);
+
+console.log(a);
+
+// Print higher value z or a 
+
+if (a<z) {
+    var i = z;
+    console.log(i);
+
+} else {
+    var i = a;
+    console.log(i);
+
+}
+
+// Animals Names
+
+var myFavoriteAnimals;
+
+console.log('The value of heros is: animales\'s names');
+
+console.log(myFavoriteAnimals);
+
+myFavoriteAnimals = ['monkey', 'horse', 'turtle'];
+
+console.log(myFavoriteAnimals);
+
+myFavoriteAnimals = myFavoriteAnimals + [', baby pig'];
```

Comment:
A cleaner (and better) solution is:

`myFavoriteAnimals.push('baby pig');`

I think this works better:)